packages/cli/src/lib/spaces/spaces.ts

```diff
@@ -0,0 +1,7 @@
+import * as Heroku from '@heroku-cli/schema'
+
+export function displayNat(nat?: Required<Heroku.SpaceNetworkAddressTranslation>) {
```

Comment:
Does the type need to be `Required` here?

Reply:
I'm adding it everywhere because our base schema isn't correct. All properties are indeed required, and many times the fact that the schema wrongfully adds the optional modifier, it will complain on function calls that doesn't allow `undefined` values. Now, I haven't checked if in this case it is really needed, but it will never hurt and it's more correct than what the base schema provides.